### PR TITLE
THEMES-1907: Sign Author Images

### PR DESCRIPTION
**Pull request by rmbrntt**
## Description
Added an additional check in `signImagesInANSObject` against `author` ANS to handle the image url as an external image to sign.

#### Jira Ticket: [THEMES-1907](https://arcpublishing.atlassian.net/browse/THEMES-1907)

# Proof
## Before
`content-api`
image

## After
`content-api`
image
`story-feed-query`
image
`author-api`
image

## Test Steps

_Add detailed test steps a reviewer must complete to test this PR_

1. Checkout this branch - `git checkout sign-author-images`
2. Query a story in debugger like `/travel/2022/12/15/get-ahead-on-holiday-travel/` http://localhost/pagebuilder/tools/debugger?_website=the-gazette&type=content-debugger&source=content-api&config=%7B%22website_url%22:%22/travel/2022/12/15/get-ahead-on-holiday-travel/%22%7D
3. Confirm the author image object now has an auth key on it.
4. Make the same query on sandbox to compare that it does not exist https://sandbox.themesinternal.arcpublishing.com/pagebuilder/tools/debugger?_website=the-gazette&type=content-debugger&source=content-api&config=%7B%22website_url%22:%22/travel/2022/12/15/get-ahead-on-holiday-travel/%22%7D

To actually test this all the way through, we need to make an update to the author blocks as they aren't expecting to pass in the resize params or ANS image that the Image component expects. That'll be a separate PR

[THEMES-1907]: https://arcpublishing.atlassian.net/browse/THEMES-1907?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Note
We duplicated resizer code into feed-components which means we should also have this logic included there.

### CODE REVIEW

**Review by vgalatro — APPROVED** [review]
LGTM!

**Comment by nschubach** [comment]
Before merging, please be sure https://github.com/WPMedia/arc-themes-components/pull/266 is merged

File: src/utils/sign-images-in-ans-object/index.js
Comment (nschubach) [inline src/utils/sign-images-in-ans-object/index.js]:
```suggestion
	if (type === "author" && image?.match(/^https?://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?$/i)) {
```

File: src/utils/sign-images-in-ans-object/index.js
Comment (nschubach) [inline src/utils/sign-images-in-ans-object/index.js]:
```suggestion
	if (type === "author" && image?.match(/^https?:\/\/(-\.)?([^\s\/?\.#-]+\.?)+(\/[^\s]*)?$\/i)) {
```

**Comment by rmbrntt** [comment]
feel free to merge this when it's good to go

**Pull request merged**
